src/server/server_send.c

```
@@ -124,3 +124,63 @@ void bc_send_to_all(const u8 *payload, int payload_len, bool reliable)
     }
 }
+
+/* Round-robin cursor across peers for fairness: the drain starts from a
+ * different peer each tick so no single peer is consistently first to claim
+ * datagram budget under load. */
+static int g_drain_cursor = 1;  /* slot 0 is the dedi host, never drained */
+
+void bc_drain_peer(int slot)
+{
+    bc_peer_t *peer = &g_peers.peers[slot];
+    if (!bc_drain_pending(&peer->drain)) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)